wannierberri/__result.py

```diff
@@ -470,59 +471,60 @@ def transform(self,sym):
         data=[sym.transform_tensor(data,rank=self.rank,TRodd=self.TRodd,Iodd=self.Iodd) for data in self.data_list]
         return KBandResult(data,self.TRodd,self.Iodd)
 
-
```

Comment:
Are the changes here only in formatting of the code? or smth else?

Reply:
There is a new utility function get_component_list(self) to KBandResults, which return the list of components (e.g. ["xx", "xy", ...]).
And the remaining is just formatting. (Sorry for mixing feature and formatting in the same PR.